src/main/java/com/minecolonies/coremod/inventory/InventoryCitizen.java

```diff
@@ -1034,13 +1034,13 @@ public void setItemStack(ItemStack itemStackIn)
      * @param itemStackIn the stack to be searched for.
      * @return true if it exists.
      */
-    public boolean hasItemStack(ItemStack itemStackIn)
+    public boolean hasItemStack(final ItemStack itemStackIn)
     {
         label19:
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor the code to remove this label and the need for it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALabelsShouldNotBeUsedCheck)